lib/index.js

```diff
@@ -39,6 +39,164 @@ Wigzo.prototype.loaded = function() {
 };
 
 /**
+ * Product Added
+ *
+ * @param {Track} track
+ */
+
+Wigzo.prototype.productAdded = function(track) {
+  var props = track.properties();
+  if (props.product_id) {
```

Comment:
we can refactor this to:

```
var productId = track.productId();
if (productId) window.wigzo.track('addtocart', productId);
```
`productId()` is nice in that it will handle casing in case someone sends `productId/product_id`
hmm any reason why you're not accepting all the other rich product data? #